### DISCUSSION TITLE
Gridappsd-2030-5 install and repo

### DISCUSSION THREAD

**Discussion by mjames-utilidata** [GRIDAPPSD/GOSS-GridAPPS-D#1760] [Installation]
Following the install instructions [here](https://pypi.org/project/gridappsd-2030-5/#description) I'm working on installing the 2030.5 server, using python 3.11, and getting build errors for gevent.  
Chasing this down a bit, it looks like the specified version of gevent is incompatible with 3.11 (see this [bug report](https://github.com/gevent/gevent/issues/1903) ).

Trying to chase this down further led me to try to clone the git repo, but the given link (https://github.com/GRIDAPPSD/gridappsd-2030_5) doesn't exist, and searching for the repo within the GRIDAPPSD org also doesn't turn up anything.

In the near term, I can try to run under a previous version of Python (maybe 3.10?), but 3.11 is over a year old and libraries such as gevent have been updated to support it.

Any help/advice would be appreciated.

P.S. Thanks for all the work on getting the 2030.5 server implementation coded!

**Comment by poorva1209** [comment]
Adding @craig8  to the discussion.

**Reply by mjames-utilidata** [reply]
Quick ping to see if there are any updates.  Thanks!
